e2e_test/sink/remote/jdbc.check.pg.slt

```diff
@@ -3,8 +3,8 @@
 query I
 select * from t_remote order by id;
 ----
-1 Alex 28208 281620391 4986480304337356800 28162.0391 2.03 28162.0391 2023-03-20 10:18:30
```

Comment:
It is because we are using stream chunk that the result gets more precise?

Reply:
No. The origin test data is wrong. It should not lost precision anyway.